x-pack/plugins/spaces/public/nav_control/nav_control.tsx

```diff
@@ -4,7 +4,6 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import './nav_control.scss';
```

Comment:
I checked the single selector that existed in this file and didn't find any instances so I just removed it.